concordium-node/src/rpc.rs

```diff
@@ -450,6 +459,10 @@ impl P2p for RpcServerImpl {
                         ConsensusIsInBakingCommitteeResponse::NotInCommittee => {
                             node_info_response::IsInBakingCommittee::NotInCommittee.into()
                         }
+                        ConsensusIsInBakingCommitteeResponse::ShutDown => {
+                            node_info_response::IsInBakingCommittee::AddedButNotActiveInCommittee
+                                .into()
+                        }
```

Comment:
It seems that this is not an accurate response. In particular, if there is no baker and the consensus is shut down, this should not be the response. Really, if the consensus is shut down then the node has no idea whether it is supposed to be in the baking committee or not, but it would generally be most accurate to reflect the status as of the last block of the chain.

Reply:
Removed the response type.